### DISCUSSION TITLE
开源史海钩沉系列 [1] RAY：分布式计算框架 | 高策

### DISCUSSION THREAD

**Discussion by giscus** [gaocegege/blog-comments#21] [General]
# 开源史海钩沉系列 [1] RAY：分布式计算框架 | 高策

江湖小虾米

https://gaocegege.com/Blog/why-do-i-like-ray

**Comment by wey-gu** [comment]
感谢，写的好棒。

**Reply by gaocegege** [reply]
这都好几年前的了😄

**Comment by Leo-Hans** [comment]
兄弟，现在RAY还值得跟进吗

**Reply by gaocegege** [reply]
还是值得了解一下的，应用场景比 20 年多很多

**Comment by pxp531** [comment]
看了大佬的kubeflow，感觉ray有点kubeflow+k8s的意思，将机器学习变成一体化和分布式的，这个理解对吗，ray的好处在哪哇 没太get到

**Reply by gaocegege** [reply]
好处在于简单吧，不需要写 YAML，只需要写 Python（理想情况下）

**Reply by pxp531** [reply]
如果只有这个优势是不是有点难发展起来🤣还没太get到ray的用处和场景

**Reply by gaocegege** [reply]
易用性我觉得是挺大的优势
